eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -205,7 +204,8 @@ public function lookup($url)
             try {
                 $this->logger->logCall(__METHOD__, array('url' => $url));
                 $urlAlias = $this->persistenceHandler->urlAliasHandler()->lookup($url);
-                $cache->set($urlAlias->id);
+                $urlAliasId = $urlAlias->id;
```

Comment:
It just an residue from cache warmup, i will change this. Also I created new jira ticket for this cache warmup feature.

Url alias cache warmup: https://jira.ez.no/browse/EZP-25773